mobile/src/main/java/rs/readahead/washington/mobile/views/activity/CameraActivity.java

```diff
@@ -74,6 +71,7 @@
 import rs.readahead.washington.mobile.views.custom.CameraGridButton;
 import rs.readahead.washington.mobile.views.custom.CameraResolutionButton;
 import rs.readahead.washington.mobile.views.custom.CameraSwitchButton;
+import rs.readahead.washington.mobile.databinding.ActivityCameraBinding;
 
 
 public class CameraActivity extends MetadataActivity implements
```

Comment:
Let's create a `BaseBindingActivity` similar to `BaseBindingFragment` and extends all activities from it.